ui/src/components/AppDetailedView.js

```diff
@@ -16,6 +16,7 @@ class AppDetailedView extends React.Component {
     return (
       <div className="app-detailed-view">
         <Header />
+        <AppDetailsToolbarContainer />
```

Comment:
I think the toolbar would look better if it was the same width as the table below.

Reply:
@craicoverflow I will update the padding on the left. In the flat the buttons are more towards the corner so I've left those for now. Let me know if you want that update. Resolving this one for now unless change to the buttons margin is requested.